src/abstractdataframe/show.jl

```diff
@@ -68,7 +68,27 @@ if VERSION < v"1.5.0-DEV.261" || VERSION < v"1.5.0-DEV.266"
     end
 end
 
-"""Return compact string representation of type T"""
```

Comment:
Add a comment to explain what's the point of having this function?

Reply:
added

Reply:
I don't see it. :-D

Reply:
ah - added. I was looking at another view on GitHub and thought you want me to expand the docstring of `compacteltype` (which I did)